microcosm_secretsmanager/loaders/base.py

```diff
@@ -0,0 +1,65 @@
+from json import loads
+from json.decoder import JSONDecodeError
+
+from boto3 import Session
+from botocore.exceptions import ClientError
+
+from microcosm_logging.decorators import logger
+
+
+@logger
```

Comment:
See below: I'm far from convinced that logging will be usable when you run this code. Maybe just print?